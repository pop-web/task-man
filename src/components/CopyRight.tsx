import Link from "@material-ui/core/Link";
import { Box, Typography } from "@material-ui/core";

const Copyright:React.FC = () => {
  return (
    <Box mt={8}>
      <Typography variant="body2" color="textSecondary" align="center">
        {"Copyright Â© "}
        <Link color="inherit" href="/">
          TASK MAN
        </Link>{" "}
        {new Date().getFullYear()}
        {"."}
      </Typography>
    </Box>
  );
};

export default Copyright;
