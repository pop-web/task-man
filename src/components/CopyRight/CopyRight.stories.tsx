import CopyRight from "./CopyRight";

export default {
  title: "CopyRight",
  component: CopyRight,
};

const Template = () => <CopyRight />;
export const Default = Template.bind({});
