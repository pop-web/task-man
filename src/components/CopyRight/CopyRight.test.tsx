import renderer from "react-test-renderer";
import CopyRight from "../CopyRight";

it("Page snapshot testing", () => {
  const component = renderer.create(<CopyRight />);
  expect(component).toMatchSnapshot();
});
