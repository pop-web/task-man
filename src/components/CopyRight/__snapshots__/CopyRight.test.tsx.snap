// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Page snapshot testing 1`] = `
<div
  className="MuiBox-root MuiBox-root-1"
>
  <p
    className="MuiTypography-root MuiTypography-body2 MuiTypography-colorTextSecondary MuiTypography-alignCenter"
  >
    Copyright Â© 
    <a
      className="MuiTypography-root MuiLink-root MuiLink-underlineHover MuiTypography-colorInherit"
      href="/"
      onBlur={[Function]}
      onFocus={[Function]}
    >
      TASK MAN
    </a>
     
    2021
    .
  </p>
</div>
`;
